docs/data/charts/overview/overview.md

```diff
@@ -7,34 +7,29 @@ packageName: '@mui/x-charts'
 
 # MUI X Charts
 
-<p class="description">A fast and extendable library of react chart components for data visualization.</p>
+<p class="description">A collection of React UI chart components for data visualization.</p>
```

Comment:
This would break the SEO query optimization otherwise #10216.

```suggestion
<p class="description">A collection of React chart components for data visualization.</p>
```

Also, "extendable" is an important differentiator for the library, not sure we should remove it.